### ISSUE TITLE
Radio button borders do not provide sufficient contrast

### ISSUE THREAD

**Issue by chlane** [vmware/clarity#4587]
[Clarity Version]
3.1.1

[Test URL]
https://clarity.design/documentation/radio

[WCAG 2.1 Guideline]
1.4.11 Non-text Contrast
(Level AA)

[OS]
Any

[User Agents/Assistive Technologies]
NA

[Component]
Radio Buttons

[Description]
Radio button borders do not provide sufficient contrast. The foreground color of the border is #b3b3b3 and the background color #fff yield a contrast ratio 2.10:1.  The contrast ratio of the border should be at least 3:1. 

This issue is related to https://jira.eng.vmware.com/browse/VUM-7307. 

See the relevant HTML/CSS below.

&lt;label class=&quot;clr-control-label ng-star-inserted&quot; for=&quot;clr-form-control-9&quot;&gt;&lt;/label&gt;

.clr-radio-wrapper input[type=radio]+label:before {
    position: absolute;
    top: .2rem;
    left: 0;
    content: &quot;&quot;;
    display: inline-block;
    height: .8rem;
    width: .8rem;
    border: .05rem solid #b3b3b3;
    border-color: var(--clr-forms-border-color,#b3b3b3);
    border-radius: 50%;
}

[User Impact]
Low vision users may not be able to perceive the radio buttons.

[Steps to Reproduce]
- Go the test URL
- Observe that the borders of the radio buttons are faint
- Test the foreground and background color combination for the radio button&#039;s border at https://www.levelaccess.com/color-contrast-checker/

[Recommendation]
The contrast ratio of the border should be at least 3:1. 

[Reporter Email] 
chlane@vmware.com
